DQM/Physics/python/topSingleLeptonDQM_cfi.py

```diff
@@ -512,11 +134,11 @@
 			rho = cms.InputTag("fixedGridRhoFastjetAll"),
     ),
     muonExtras = cms.PSet(
-      select     = cms.string(looseMuonCut + " & pt>20 & abs(eta)<2.1"),
-			isolation  = cms.string(looseIsoCut),
+      select     = cms.string(tightMuonCut + " & pt>20 & abs(eta)<2.4"),
+      isolation  = cms.string(looseIsoCut),
     ),
     jetExtras = cms.PSet(
-			jetCorrector = cms.InputTag("dqmAk4PFCHSL1FastL2L3Corrector"), #Use pak4PFCHSL1FastL2L3Residual for data!!!
+			jetCorrector = cms.InputTag("ak4PFCHSL1FastL2L3Corrector"), #Use pak4PFCHSL1FastL2L3Residual for data!!!
```

Comment:
This is breaking again the issue found in https://github.com/cms-sw/cmssw/issues/22996
and fixed in https://github.com/cms-sw/cmssw/pull/28156
Please revert the change and test it getting the Tier0 configuration as explained in https://its.cern.ch/jira/browse/CMSTZ-427
Thanks

Reply:
Local test with "topDQM_production_cfg.py" after changed to "dqmAk4PFCHSL1FastL2L3Corrector" ; but got error, not define "dqmAk4PFCHSL1FastL2L3Corrector". 
So, added line: process.load("DQMOffline.JetMET.jetMETDQMOfflineSource_cff")

----- Begin Fatal Exception 27-Mar-2020 06:43:11 CET-----------------------
An exception of category 'ProductNotFound' occurred while
   [0] Processing  Event run: 1 lumi: 31 event: 3005 stream: 0
   [1] Running path 'p'
   [2] Calling method for module ChainedJetCorrectorProducer/'dqmAk4PFCHSL1FastL2L3Corrector'
Exception Message:
Principal::getByToken: Found zero products matching all criteria
Looking for type: reco::JetCorrector
Looking for module label: ak4PFCHSL1FastjetCorrector
Looking for productInstanceName:

   Additional Info:
      [a] If you wish to continue processing events after a ProductNotFound exception,
add "SkipEvent = cms.untracked.vstring('ProductNotFound')" to the "options" PSet in the configuration.

Can you see why?

Reply:
No sorry, I don't see why, I am afraid you should dig into the config
Thanks